controllers/digital-funding/views/strand-1.njk

```diff
@@ -0,0 +1,173 @@
+
+                    <div class="content-sidebar__secondary">
+                        {{ getStarted(
+                            url = '/funding/programmes/digital-funding/strand-1/eligibility',
+                            accent = pageAccent
+                        ) }}
+                    </div>
+                </div>
+            </section>
+
+
+            {% call contentBox(pageAccent, id='segment-2') %}
+                <h2>Success factors</h2>
+
+                <p>Successful proposals will:</p>
+
+                <ul>
+                    <li>Clearly demonstrate either the potential to scale impact at low cost, or clearly demonstrate potential to use digital to bring substantial benefits to a specific group of people.</li>
+                    <li>Be driven by teams that combine knowledge of a social issue with experience of digital delivery.</li>
+                    <li>Relate to a product or service that is far enough through its development process that real users are currently benefiting from it.</li>
+                    <li>Be able to provide at least some evidence of impact upon those real users.</li>
+                    <li>Have a theory of change or similar document showing how the product or service will achieve social impact.</li>
+                    <li>Show an understanding of the challenge of sustaining impact after the ending of Big Lottery Fund support.</li>
+                </ul>
+
+                <p>We are <strong>not</strong> looking for:</p>
+                <ul>
+                    <li>Products or services that are not yet mature enough to be benefiting at least a few people</li>
+                    <li>Products of services that cannot show considerable evidence of user research in their composition</li>
+                    <li>Products or services that duplicate existing services with no substantial innovation</li>
+                </ul>
+
+            {% endcall %}
+            {% call contentBox(pageAccent, id='segment-2') %}
+                <h2>Eligibility criteria</h2>
+
+                <ul>
+                    <li>Your organisation must have a clear social impact mission that is embedded in your governing documents and your working culture.</li>
+                    <li>Your organisation must have been incorporated after the year 2000</li>
+                    <li>You must currently operate a product or service that is functional and being used by the kinds of beneficiaries you plan to serve in future</li>
+                </ul>
+
+                <p>You can apply if you’re a:</p>
+
+                <ul>
+                    <li>voluntary and community organisation</li>
+                    <li>registered charity</li>
+                    <li>social enterprise</li>
+                    <li>group of organisations, as long as they are led by a voluntary and community organisations or social enterprise</li>
+                    <li>Community interest companies (with two or more directors).</li>
```

Comment:
Same as above (maybe this duplicated copy should be shared in macros or something between these pages?)